packages/documentation/src/content/docs/integration/playground/overview.mdx

```diff
@@ -37,7 +37,7 @@ This environment will set up a playground where you can use the Rafiki Admin API
 
```

Comment:
This comment is re: the line above it (line 36). Can you change "use the Rafiki Admin APIs" to "use the GraphQL Admin APIs" so there's no confusion between the APIs and the Rafiki Admin?